deepchecks/vision/utils/transformations.py

```diff
@@ -118,24 +144,50 @@ def get_robustness_augmentations(cls, data_dim: Optional[int] = 3):
         return augmentations
 
 
-def get_transforms_handler(transforms):
+class TorchTransformations(AlbumentationsTransformations):
```

Comment:
TorchTransformations should have a different get_robustness_augmentations than AlbumentationsTransformations. 
Ideally, if it's classification we want the same things we had in AlbumentationsTransformations, and if it's any other task we want them excluding ShiftScaleRotate. Can we do that?

Reply:
Also, why not use native torch augmentations for TorchTransformations?